### ISSUE TITLE
Add content to Readme.md and make a Contributing.md

### ISSUE THREAD

**Issue by Gupta-Anubhav12** [gita/Bhagavad-Gita-App#91]
Adding detailed overview of project and linking required docs

**Comment by Buddhad** [comment]
I want to work on this issue please assign me .

**Comment by samanyougarg** [comment]
@Buddhad still interested in taking this on?

**Comment by Buddhad** [comment]
Yeah, I am interested and I did `PR` on this issue. Could you please check that and if some things need to change then please let me know. @samanyougarg

**Comment by TiwariAbhishek23** [comment]
I think it s still pending; if yes then please allow me to complete it and be a part of this initiative

**Comment by darkbits018** [comment]
i would like to contribute to this please assign it to me
